seccomp.go

```diff
@@ -310,7 +310,7 @@ func GetArchFromString(arch string) (ScmpArch, error) {
 		return ArchPARISC64, nil
 	case "riscv64":
 		return ArchRISCV64, nil
-	case "loongarch64":
+	case "loong64", "loongarch64":
```

Comment:
From what I see, the official name is `loongarch64`. While I find the `arch` substring redundant, it is what it is. For example, see https://github.com/seccomp/libseccomp/blob/7db46d72f13c172b290818f624c2966bd0db5677/src/arch.c#L210 in which we only have `loongarch64` and not `loong64`.

Reply:
OTOH the official Go name is loong64 which I believe Go library should recognize.